setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup
+setup(
+  name = 'coala-decorators',
+  packages = ['coala-decorators'],
+  version = '0.1',
+  description = 'All the coala decorators',
```

Comment:
i have changed the commit message. what do you think would be a better description?